package.json

```diff
@@ -42,7 +42,7 @@
     "@web/test-runner": "^0.19.0" ,
     "@web/test-runner-playwright": "^0.11.0",
     "@web/test-runner-puppeteer": "^0.17.0",
-    "@web/test-runner-saucelabs": "^0.11.2",
+    "@web/test-runner-saucelabs": "^0.12.0",
```

Comment:
They now also released [`0.13.0`](https://github.com/modernweb-dev/web/releases/tag/%40web%2Ftest-runner-saucelabs%400.13.0), which includes support for Sauce Connect 5.

Reply:
Thanks, updated.